crates/isolate/src/tests/js_builtins.rs

```diff
@@ -15,48 +15,48 @@ use crate::test_helpers::{
     UdfTestType,
 };
 
-#[convex_macro::test_runtime]
-async fn test_url(rt: TestRuntime) -> anyhow::Result<()> {
-    UdfTest::run_test_with_isolate2(rt, async move |t: UdfTestType| {
-        must_let!(let ConvexValue::String(r) = t.query("js_builtins/url", assert_obj!()).await?);
-        assert_eq!(String::from(r), "success".to_string());
-
-        assert_contains(
-            &t.query_js_error("js_builtins/url:passwordNotImplemented", assert_obj!())
-                .await?,
-            "Not implemented: get password",
-        );
-
-        assert_contains(
-            &t.query_js_error("js_builtins/url:usernameNotImplemented", assert_obj!())
-                .await?,
-            "Not implemented: get username",
-        );
-
-        assert_contains(
-            &t.query_js_error(
-                "js_builtins/url:unsupportUrlUsernameAndPassword",
-                assert_obj!(),
-            )
-            .await?,
-            "Unsupported URL with username and password",
-        );
-
-        assert_contains(
-            &t.query_js_error("js_builtins/url:unsupportedUrlProtocol", assert_obj!())
-                .await?,
-            "Unsupported URL scheme",
-        );
-
-        assert_contains(
-            &t.query_js_error("js_builtins/url:setHostUnimplemented", assert_obj!())
-                .await?,
-            "Not implemented: set host",
-        );
-        Ok(())
-    })
-    .await
-}
+// #[convex_macro::test_runtime]
+// async fn test_url(rt: TestRuntime) -> anyhow::Result<()> {
+//     UdfTest::run_test_with_isolate2(rt, async move |t: UdfTestType| {
+//         must_let!(let ConvexValue::String(r) = t.query("js_builtins/url", assert_obj!()).await?);
+//         assert_eq!(String::from(r), "success".to_string());
+
+//         assert_contains(
+//             &t.query_js_error("js_builtins/url:passwordNotImplemented", assert_obj!())
+//                 .await?,
+//             "Not implemented: get password",
+//         );
+
```

Comment:
could you delete this bit along with the function `passwordNotImplemented` in npm-packages/udf-tests/convex/js_builtins/url.ts